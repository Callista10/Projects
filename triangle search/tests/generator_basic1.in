-N=5 -mindist=2 -rseed=2