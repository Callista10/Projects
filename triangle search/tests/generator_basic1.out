45.60, 44.78
-44.34, -41.51
33.55, 23.60
16.97, -19.19
10.59, 10.68
